cmd/accountcontextparams.go

```diff
@@ -32,18 +32,26 @@ func (p *AccountContextParams) BindFlags(cmd *cobra.Command) {
 func (p *AccountContextParams) SetDefaults(ctx ActionCtx) error {
 	config := GetConfig()
 	if p.Name != "" {
+		// if specified, sync the context
 		err := config.SetAccountTemp(p.Name)
 		if err != nil {
 			return err
 		}
 		ctx.StoreCtx().Account.Name = p.Name
-		return nil
 	} else {
 		if config.Account != "" {
 			ctx.StoreCtx().Account.Name = config.Account
 			p.Name = config.Account
 		}
 	}
+	if p.Name != "" {
```

Comment:
why not combine with the earlier if?

Reply:
the value tries to set from 2 different places. then that check, verifies it was set.